tests/spread/general/patchelf/task.yaml

```diff
@@ -0,0 +1,4 @@
+summary: build a snap with chisel slices
+
+execute: |
+  run_snapcraft pack
```

Comment:
For tests that just pack and don't have any custom stuff, maybe it'd be worth using variants?

Reply:
good idea! I'll do that for sure